ci/gpu/build.sh

```diff
@@ -57,7 +57,7 @@ conda list
 # Fixing Numpy version to avoid RuntimeWarning: numpy.ufunc size changed, may
 # indicate binary incompatibility. Expected 192 from C header, got 216 from PyObject
 conda install "cudatoolkit=$CUDA_REL" \
-              "cupy>=6.5.0" "numpy=1.16.4" \
+              "cupy<7.2" "numpy=1.16.4" \
```

Comment:
Would add the `a` bit to avoid getting pre-releases as well.

```suggestion
              "cupy<7.2.0a0" "numpy=1.16.4" \
```